### ISSUE TITLE
WND Configuration article redesign.

### ISSUE THREAD

**Issue by voroyam** [owncloud/docs#3265]
## WHAT Needs to be Documented?
we need to have a look at all the warnings, notes, tips, cautions in the document and collect them in a separate section.

## WHERE Does This Need To Be Documented (Link)?
https://github.com/owncloud/docs/blob/master/modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

## WHY Should This Change Be Made?
https://github.com/owncloud/docs/pull/3249#issuecomment-791328911

basically the reason is - warnings should be short and rare.

What we have now looks like a rug that has been patched too many times.
